enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1408 @@
+a `maintenanceSchedule` stanza should also be provided to configure the strategy
+(if it is not, the policy is functionally identical to `Restrictive`).
+
+```yaml
+kind: ChanageManagementPolicy
+spec:
+  strategy: MaintenanceSchedule
+ 
+  maintenanceSchedule:
+
+    # Specifies a reoccurring permissive window. 
+    permit:  
+      
+      # If null, all dates are permitted and only exclude constrains permissive windows.
+      recurrence: 
+        # "frequency" is a discriminant for the unioned types which follow.
+        frequency: Yearly | Monthly | Weekly | Daily
+        
+        # Required when frequency=Daily
+        daily:
+          interval: <int> # How many days should pass before the next occurrence 0 < x < 731
+
+        # Required when frequency=Weekly
+        weekly:
+          daysOfWeek: 
+          - Monday
+          - Tuesday
+          - Wednesday
+          interval: <int> # How many weeks should pass before the next occurrence 0 < x < 27
+
+        # Required when frequency=Monthly
+        monthly:
+          # "by" is a discriminant for the unioned types which follow.
+          by: Day | Date
+          date:
+            datesOfMonth: 
+            - <int> # List of dates 0 < x < 32
+            interval: <int> # How many months should pass before the next occurrence 0 < x < 12
+          day:
+            days:
+            - weekOfMonth: First | Second | Third | Fourth | Fifth | Last
+              dayOfWeek: Monday | Tuesday | Wednesday | ... | Sunday
+            interval: <int> # How many months should pass before the next occurrence 0 < x < 12
+
+        # Required when frequency=Yearly
+        yearly:
+          # "by" is a discriminant for the unioned types which follow.
+          by: Day | Date
+          date:
+            datesOfMonth: 
+            - <int> # List of dates 0 < x < 32
+            month: January | February | ... | December
+          day:
+            days:
+            - weekOfMonth: First | Second | Third | Fourth | Fifth | Last
+              dayOfWeek: Monday | Tuesday | Wednesday | ... | Sunday
+            month: January | February | ... | December
+      
+      # Given the identification of a date by a recurrence rule, at what time (always UTC) can the 
+      # permissive window begin. "00:00" if unset.
+      startTime: <time-of-day|null>
+      
+      # Given the identification of a date by a recurrence rule, after what offset from the startTime should
+      # the permissive window close. This can create permissive windows within days that are not
+      # identified in the rule. For example, if "weekly on Saturday" is specified as a rule,
+      # startTime="20:00", duration="8h" would permit material change initiation starting
+      # each Saturday at 8pm and continuing through Sunday 4am (all times are UTC). The default
+      # duration is 24:00-startTime (i.e. to the end of the day).
+      duration: <duration|null>

```

Comment:
Given API conventions suggest that durations are difficult, perhaps we want to consider this to be a `durationMinutes` and be an `int32`. Will want to double check other APIs across openshift before we make a decision